src/libraries/System.Data.OleDb/src/OleDbCommand.cs

```diff
@@ -818,7 +818,6 @@ private int ExecuteCommandText(out object executeResult)
             RuntimeHelpers.PrepareConstrainedRegions();
             try
             {
-                // TODO-NULLABLE: Code below seems to assume that bindings will always be non-null
```

Comment:
this didn't seem possible to me by quickly glancing at the code as we seem to be initializing it before executing (cc: @roji to double check)

Reply:
Seems right, not sure what I had in mind here.